src/source-netmap.c

```diff
@@ -234,17 +286,39 @@ int NetmapGetRSSCount(const char *ifname)
 #else
     if (ioctl(fd, NIOCGINFO, &nm_req) != 0) {
 #endif
-        SCLogError(SC_ERR_NETMAP_CREATE, "Couldn't query netmap for info about %s, error %s",
-                ifname, strerror(errno));
+        SCLogError(SC_ERR_NETMAP_CREATE, "Couldn't query netmap for HW rings count of %s, error %s",
+                base_name, strerror(errno));
         goto error_fd;
     };
 
+        /* check for asymmetrical TX/RX queue counts on interface
```

Comment:
nit: indentation of comments should match that of `if` block

Suggest refactoring this code block to minimize duplication

```
#if USE_NET_NETMAP_API
rx_rings = req.nr_rx_rings;
int tx_rings = req.nr_tx_rings;
#else
rx_rings = nm_req.nr_rx_rings;
int rx_rings = nm_req.nr_tx_rings;
#endif
if (rx_rings != tx_rings) {
 ....
}
```